recipes/oommf/build.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+TCLTKVERSION=8.5
+CONDA_ROOT_PATH=$(conda info --root)
+
+# Compile OOMMF
+export OOMMF_TCL_CONFIG=${PREFIX}/lib/tclConfig.sh
+export OOMMF_TK_CONFIG=${PREFIX}/lib/tkConfig.sh
+cd oommf
+tclsh${TCLTKVERSION} oommf.tcl pimake distclean
```

Comment:
Ah, I see - thank you. 

In that case, I prefer to stick to hardcoding tclsh8.5 in our execution script: our users are likely to be beginners with conda, and will not be able to give higher priority to conda-forge over conda: if they can install something that looks like anaconda, and then install oommf using a line we give them (such as `conda install -c conda-forge oommf`), then that is already an achievement. If it then fails in a subtle way (because the have the conda tk and not conda-forge tk), that would be counterproductive.

I hope that makes sense and is agreeable?